lib/core/enums/amplitude_events.dart

```diff
@@ -47,6 +47,18 @@ enum AmplitudeEvents {
   editPhoneNumberSaveClicked('edit_phone_number_save_clicked'),
   editEmailSaveClicked('edit_email_save_clicked'),
 
+  // Gratitude Goal
+  gratitudeGoalStartClicked('gratitude_goal_start_clicked'),
+  gratitudeGoalContinueClicked('gratitude_goal_continue_clicked'),
```

Comment:
Most of the times continue buttons have the same event type I think :) 
If we do a separate event per button we will reach the limit quicker (max 2000 event types)

Reply:
I'll ask Tine because we sat down for these events together. Right now we are around 370ish events and while there is a "generic continue" event I do also see specific continue events in other features.